lab-teddy/README.md

```diff
@@ -0,0 +1,47 @@
+# Description:
```

Comment:
Remember read me's also describe the actual app, not just the installs of the dependencies.